GATEKEEPERS.md

```diff
@@ -0,0 +1,27 @@
+# rust-vmm gate keeping
+
+Together with repositories code owners, the gate keepers team main
```

Comment:
Grammar nits: "repository code owners" or "repositories' code owners", and "gate keepers team's main role"

Reply:
Fixed now, thanks.